components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/dao/LocalClaimDAO.java

```diff
@@ -221,6 +221,51 @@ public void updateLocalClaim(LocalClaim localClaim, int tenantId) throws ClaimMe
         }
     }
 
+    /**
+     * Update attribute claim mappings related to tenant id and domain.
+     *
+     * @param localClaimList  list of local claims
+     * @param tenantId        Tenant Id
+     * @param userStoreDomain Domain name
+     * @throws ClaimMetadataException If an error occurred while updating local claims
+     */
+    public void updateLocalClaimMappings(List<LocalClaim> localClaimList, int tenantId, String userStoreDomain)
+            throws ClaimMetadataException {
+
+        Connection connection = IdentityDatabaseUtil.getDBConnection(true);
+
+        try {
+            // Start transaction
```

Comment:
Correct commenting format is // Comment.  
A Comment always needs to be end with a full stop.

Reply:
Addressed with commit abcc28ca2bde2f2eea597841ec5000f0cddd7202